client/js/_admin/components/lti_installs/account_install.jsx

```diff
@@ -0,0 +1,62 @@
+import React                          from 'react';
+import _                              from 'lodash';
+import {
+  createExternalToolAccounts,
+  deleteExternalToolAccounts,
+ } from '../../../libs/canvas/constants/external_tools';
+
+export default function AccountInstall(props) {
```

Comment:
you should abstract the install function out to a parent class and then pass it in as a prop.